File: Fonksiyonlar.cpp
// Fonksiyon Kullanýmý
#include <iostream>
using namespace std;

//Fonksiyonumuz
int Topla (int sayi1, int sayi2)
{
	int sonuc = sayi1 + sayi2;
	return sonuc;
}	
	int a,b;


int main()
{
	
	
	cout << "1. sayiyi giriniz : ";
	cin >> a;

	cout << "2. sayiyi giriniz : " ;
	cin >> b;
	
	int sonuc = Topla(a,b);
	cout << "Toplam =  " << sonuc << endl;
}

File: Java/src/Fonksiyonlar.java
import java.util.Scanner;
public class Fonksiyonlar 
{
	public static void main(String[] args)
	{
		int a,b;
		Scanner scn = new Scanner(System.in);
		
		System.out.println("1.sayi : ");
		a = scn.nextInt();
		
		System.out.println("2.sayi : ");
		b = scn.nextInt();
		
		int sonuc = Topla (a,b);
		System.out.println("Toplam : " +sonuc);

	}
	
	public static int Topla (int sayi1,int sayi2)
	{
		int sonuc = sayi1 + sayi2;
		return sonuc;
	}

}
